config/initializers/npm.rb

```diff
@@ -0,0 +1 @@
+system 'npm install --prefix lib/asset_grabber'
```

Comment:
I'm not sure this is the appropriate place to initialize the node packages with Rails, it was a best guess. So any suggestions are welcome.